uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Reminder.php

```diff
@@ -131,14 +131,19 @@ private static function _extractPreviousAssignee(&$assignees, $previousAssigneeI
    * Gets the full list of the recipients of the reminder email
    * Makes sure there are no duplicates or no empty emails in the list
    *
-   * @param array $activityContacts
+   * @param array $contacts
    * @param array $previousAssignee
    * @return array
    */
-  private static function _reminderRecipients($activityContacts, $previousAssignee) {
-    return array_filter(array_unique(array_merge(
-                            $activityContacts['assignees']['emails'], $activityContacts['source']['emails'], [$previousAssignee['email']]
-    )));
+  private static function _reminderRecipients($contacts, $previousAssignee) {
+    $hasSources = !empty($contacts['source']['emails']);
```

Comment:
`$assignees = !empty($contacts['assignees']['emails']) ?: [];` 

would lead to $assignees being `true`. 

`$assignees = !empty($contacts['assignees']['emails']) ? $contacts['assignees']['emails'] : [];`

Is not bad, but my main reason for moving the `empty` check into a different line was to keep the line length short and make it more readable.